code/modules/vore/appearance/sprite_accessories_vr.dm

```diff
@@ -221,6 +221,82 @@
 	do_colouration = 1
 	color_blend_mode = ICON_MULTIPLY
 
+/datum/sprite_accessory/ears/xeno_head_drone
+	name = "drone dome"
+	icon_state = "xeno_drone"
+	gender = NEUTER
+		
+/datum/sprite_accessory/ears/xeno_head_sentinel
+	name = "sentinal dome"
+	icon_state = "xeno_sentinel"
+	gender = NEUTER
+	
+/datum/sprite_accessory/ears/xeno_head_queen
+	name = "queen dome"
+	icon_state = "xeno_queen"
+	gender = NEUTER
+
+/datum/sprite_accessory/ears/xeno_head_hunter
+	name = "hunter dome"
+	icon_state = "xeno_hunter"
+	gender = NEUTER
+
+/datum/sprite_accessory/ears/xeno_head_praetorian
+	name = "Praetorian dome"
+	icon_state = "xeno_praetorian"
+	gender = NEUTER
+
+/datum/sprite_accessory/ears/xeno_head_drone_black
+	name = "black drone dome"
+	icon_state = "bxeno_drone"
+	gender = NEUTER
+		
+/datum/sprite_accessory/ears/xeno_head_sentinel_black
+	name = "black sentinal dome"
+	icon_state = "bxeno_sentinel"
+	gender = NEUTER
+	
+/datum/sprite_accessory/ears/xeno_head_queen_black
+	name = "black queen dome"
+	icon_state = "bxeno_queen"
+	gender = NEUTER
+
+/datum/sprite_accessory/ears/xeno_head_hunter_black
+	name = "black hunter dome"
+	icon_state = "bxeno_hunter"
+	gender = NEUTER
+
+/datum/sprite_accessory/ears/xeno_head_praetorian_black
+	name = "black praetorian dome"
+	icon_state = "bxeno_praetorian"
+	gender = NEUTER
+//need to figure out coloring stuff
+
+/datum/sprite_accessory/ears/xeno_head_drone_color
+	name = "colorable drone dome"
+	icon_state = "cxeno_drone"
+	gender = NEUTER
+/*		
+/datum/sprite_accessory/ears/xeno_head_sentinel_color
+	name = "colorable sentinal dome"
+	icon_state = "cxeno_sentinel"
+	gender = NEUTER
+*/
```

Comment:
The comment above makes me think this commenting out should end at line 299 instead of 284.